src/features/NotificationsPanel/useNotifications.spec.tsx

```diff
@@ -28,97 +27,71 @@ describe('useNotifications', () => {
     expect(result.current.notifications).toEqual(fNotifications);
   });
 
-  it('uses a valid data model', () => {
-    const createActions = jest.fn();
-    renderUseNotifications({ createActions });
-    expect(createActions).toHaveBeenCalledWith(LSKeys.NOTIFICATIONS);
-  });
-
   it('current notification', () => {
     const notification = { ...fNotifications[0], dismissed: false, dateDismissed: undefined };
-    const { result } = renderUseNotifications({
-      notifications: [notification],
-      createActions: jest.fn().mockImplementation(() => ({
-        update: jest.fn()
-      }))
-    });
+    const { result } = renderUseNotifications({ notifications: [notification] });
     expect(result.current.currentNotification).toBe(notification);
   });
 
   it('displayNotification calls model.createWithID', () => {
```

Comment:
Can we rename all tests that now uses dispatch instead of model?

Reply:
Removed. Opted for the simplest renaming since we should be deleting the hooks soon.